Makefile

```diff
@@ -110,12 +110,16 @@ C_FILES       := $(foreach dir,$(SRC_DIRS) $(ASSET_BIN_DIRS),$(wildcard $(dir)/*
 S_FILES       := $(foreach dir,$(ASM_DIRS),$(wildcard $(dir)/*.s))
 O_FILES       := $(foreach f,$(S_FILES:.s=.o),build/$f) \
                  $(foreach f,$(C_FILES:.c=.o),build/$f) \
-                 $(foreach f,$(wildcard baserom/*),build/$f.o)
+                 $(foreach f,$(wildcard baserom/*),build/$f.o) \
+                 $(shell awk -F"\"" '/_reloc.o/ { print $$2 }' spec )
 
 # Automatic dependency files
-# (Only asm_processor dependencies are handled for now)
+# (Only asm_processor dependencies and reloc dependencies are handled for now)
 DEP_FILES := $(O_FILES:.o=.asmproc.d)
 
+RELOC_DEPS    := build/reloc_deps.d
+$(shell awk -F\" '/name "ovl/ { flag=1; out=null; dep=null } /include/ && flag && !/reloc.o/ { dep=dep " " $$2 } /_reloc.o/ { out=$$2 } /endseg/ && out && flag { flag=0; printf "%s:%s\n\n", out, dep }' spec > $(RELOC_DEPS))
```

Comment:
> If you make the built spec a prerequisite, wouldn't it rebuild every reloc file every time the spec is changed? That seems a waste.
Considering it takes 0.3 seconds to generate every reloc, doesn't seems like a huge tradeoff

Reply:
For Fado, maybe. as I can't vouch for.

Reply:
For reference, reading the spec and getting the objects for a given reloc appears to take less than 0.5ms with my WIP program, which is very poorly optimized. So I'm personally not worried about the impact it would have on the build, tho I understand the concern.

Reply:
I don't think the tool should read the spec itself at all. The tool itself shouldn't depend on the particular build system we're using.

Reply:
Unrelated to the whole discussion (github doesn't allow me to put a new comment here...):
The makefile has a symbol `SPEC`, which expands to `spec`. You should be using that imo